popup.html

```diff
@@ -1,6 +1,7 @@
 <!doctype html>
 <html>
   <head>
+    <meta charset="utf-8"/>
```

Comment:
This is not strictly relevant to the rest of the patch but firefox gave me errors if I didn't include this.